delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -291,6 +294,13 @@ class ResourcesRoutesSpec extends BaseRouteSpec {
       }
     }
 
+    "fetch a resource original payload with metadata" in {
```

Comment:
It would be nice to have some tests:
* To validate that we are forbidden to fetch the source (annotated or not) when we don't have the permission
* To check that we have a 404 when the resource does not exist
* To check we can combine annotate with the rev and tag parameters